mlir/include/mlir/Dialect/Tosa/IR/TosaTypesBase.td

```diff
@@ -74,6 +78,16 @@ def Tosa_QuantizedInt : AnyTypeOf<[Tosa_QuantizedType<"uint8", [8], 0>,
                                    Tosa_QuantizedType<"int16", [16, 0], 1>,
                                    Tosa_QuantizedType<"int32", [32, 0], 1>]>;
 
+//===----------------------------------------------------------------------===//
+// Floating-point types.
+//===----------------------------------------------------------------------===//
+def Tosa_Float : AnyTypeOf<[F32,
+                            F16,
+                            BF16]>;
+
+def Tosa_F8 : AnyTypeOf<[F8E4M3FN,
```

Comment:
It seems https://github.com/llvm/llvm-project/pull/127730 no longer relies on this addition, and I can't see `Tosa_F8` or `Tosa_Float` being used in this patch, are they still needed?

Reply:
I don't think still needed.

Reply:
Done